subcommand/server-acl-init/command_test.go

```diff
@@ -1221,5 +1293,57 @@ func getBootToken(t *testing.T, k8s *fake.Clientset, prefix string) string {
 	return string(bootToken)
 }
 
+// generateServerCerts generates Consul CA
+// and a server certificate and saves them to temp files.
+// It returns file names in this order:
+// CA certificate, server certificate, and server key.
+// Note that it's the responsibility of the caller to
+// remove the temporary files created by this function.
```

Comment:
you could also return a function that the callers can run in a defer that cleans up the files:
```go
func() a (string, func()) {
  return "a", func() {
    os.Remove("a")
  }
}

file, cleanup := a()
defer cleanup()
```

Reply:
Oh, that's an awesome idea. I'll do that!